.github/workflows/test_ert_with_flow.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 env:
   UV_FROZEN: true
+  OMP_NUM_THREADS: 1
```

Comment:
If this is meant for Flow, it should already be handled by the Flow forward model step.